tests/omise/CapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php require_once dirname(__FILE__).'/TestConfig.php';
+
+class CapabilitiesTest extends TestConfig {
+  /**
+   * OmiseCapabilities class must be contain some method below. 
+   *
+   */
+  public function testMethodExists() {
+    $this->assertTrue(method_exists('OmiseCapabilities', 'retrieve'));
+    $this->assertTrue(method_exists('OmiseCapabilities', 'reload'));
+  }
+
+  /**
+   * Assert that a capabilities object is returned after a successful retrieve.
+   *
+   */
+  public function testRetrieveOmiseCapabilitiesObject() {
+    $capabilities = OmiseCapabilities::retrieve();
+
+    $this->assertArrayHasKey('payment_backends', $capabilities);
+    $this->assertInternalType('array', $capabilities['payment_backends']);
+  }
+
+  
+  /**
+   * Assert that a capabilities object is returned after a successful reload.
+   *
+   */
+  public function testReload() {
+    $capabilities = OmiseCapabilities::retrieve();
+    $capabilities->reload();
+
+    $this->assertArrayHasKey('payment_backends', $capabilities);
+    $this->assertInternalType('array', $capabilities['payment_backends']);
+  }
+}

```

Comment:
There is also this comment but somehow it doesn't show on Slack..
Let me repost here:

@jonrandy @jacstn Hi, I finished review.
There are some part that need to apply code styling (those brackets { }, space between string, naming convention - no _ prefix for private method).

You can try merge or rebase master branch.
I've added PHP Code Sniffer already. After merge, can run ./vendor/bin/phpcs to see the result.

Reply:
Strongly disagree on no underscore prefixes for `private` and `protected` methods. It makes the code way easier to read

Reply:
I don't find it as an obstacle, guess it depends on individual style this one. 
So I'll just follow PSR-2 and the current code style for now.